go.mod

```diff
@@ -3,63 +3,119 @@ module github.com/bitrise-io/bitrise-step-analytics
 go 1.24
 
 require (
-	cloud.google.com/go/pubsub v1.19.0
+	cloud.google.com/go/pubsub v1.49.0
```

Comment:
Version 1.50 is the latest, but that marks everything in this v1 package deprecated and wants us to migrate to v2.